components/hab/install.sh

```diff
@@ -202,7 +230,37 @@ validate_target() {
   fi
 }
 
-download_archive() {
+download_packages_chef_io_archive() {
+  need_cmd mv
+
+  _version="${version:-latest}"
+
+  if [ "$_version" == "latest" ]; then
+    url="${PCIO_ROOT}/${channel}/latest/habitat/hab-${target}.${ext}"
+  else 
+    url="${PCIO_ROOT}/files/habitat/${version}/hab-${target}.${ext}"
+  fi
+  
+  dl_file "${url}" "${workdir}/hab-${version}.${ext}"
```

Comment:
Should we use _version here and below?  Otherwise won't we create this with the file name `hab-.EXT`?

Reply:
Good catch, thanks!